programs/fund-sender/src/utils/state.rs

```diff
@@ -11,6 +12,8 @@ pub struct GenericStateInput {
     pub update_authority: Pubkey,
     // the account of the climate token
     pub destination_account: Pubkey,
+    // the hold account of retired climate token
+    pub certificate_vault: Pubkey,
```

Comment:
Does the updateState instruction need updating now that this is added?

Reply:
The updateState instruction is updated accordingly. A new function `updateCertificateVault()` is also added on the client side